lib/merchant_repository.rb

```diff
@@ -0,0 +1,56 @@
+require 'CSV'
+# require './lib/merchant'
+
+class MerchantRepository
+  attr_reader :all_merchants
+
+  def initialize(path)
+    @all_merchants = create_merchants(path)
+  end
+
+  def create_merchants(path)
+  @merchants = CSV.foreach(merchants, headers: true, header_converters: :symbol).map do |merchant|
```

Comment:
your path argument should be passed into the first argument of your foreach method. It will go to the correct CSV through the path you provide. Not sure if merchants was originally pointing to that, but looks like it changed.